src/Makefile

```diff
@@ -2,10 +2,13 @@
 # Use `FC=<compiler> ... make` to select compiler
 # Use `DEBUG=1 ... make` for a debug build
 # Use `NETCDF=1 ... make` to build with NetCDF using `nf-config`
+# compile with either FC=ifort or FC=gfortran
 
 # Compiler
 FC ?= gfortran
 ifeq ($(FC), f77)  # override possible Make default
+  FC := ifort 
+else ifeq ($(FC),gfortran)
   FC := gfortran
```

Comment:
Looks like it would do nothing?

Reply:
I agree, not needed.

Reply:
@zmoon @drnimbusrain When i revert back these changes, FC shows as gfortran and i get build error:
Fatal Error: File 'netcdf.mod' opened at (1) is not a GNU Fortran module file
make: *** [canopy_ncf_io_mod.o] Error 1
I think we need these changes (or FC to be assigned ifort somewhere)

Reply:
Agree on this